dags/sentinel-2_data_transfer.py

```diff
@@ -23,8 +23,12 @@
 from airflow.contrib.hooks.aws_sns_hook import AwsSnsHook
 from airflow.hooks.S3_hook import S3Hook
 
-AFRICA_CONN_ID = "deafrica-prod-migration"
-US_CONN_ID = "deafrica-migration_us"
+# The following connection is based on svc-deafrica-prod-eks-s2-data-transfer
+# which is in deafrica account
+US_CONN_ID = "deafrica-prod-eks-s2-data-transfer"
+# The following connection is based on svc-deafrica-sentinel-2-bucket-write
+# user in the PDS account
+AFRICA_CONN_ID = "deafrica-sentinel-2-bucket-write"
```

Comment:
can this variables define in one place - under `infra/variables.py` file and imported here. This will help

- reduce duplication every where
- maintenance

Reply:
👍

Reply:
Good suggestion, I moved s2 connections to variables.py and import it from there.